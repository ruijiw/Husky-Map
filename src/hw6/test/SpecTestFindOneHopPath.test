# Find a path using the LOLDictionary dataSet.
# This test uses both the parser and BFS algorithm in the MarvelPaths
# class and applies to a targeted dataset. This test is for one
# character have one hop("edge") path to another character.

LoadGraph dictionary LOLDictionary.tsv

FindPath dictionary TEEMO LEBLANC

FindPath dictionary VI TEEMO

FindPath dictionary TEEMO JINX

FindPath dictionary JINX SONA