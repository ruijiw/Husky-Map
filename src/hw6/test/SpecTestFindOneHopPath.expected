# Find a path using the LOLDictionary dataSet.
# This test uses both the parser and BFS algorithm in the MarvelPaths
# class and applies to a targeted dataset. This test is for one
# character have one hop("edge") path to another character.

loaded graph dictionary

path from TEEMO to LEBLANC:
TEEMO to LEBLANC via MID

path from VI to TEEMO:
VI to TEEMO via JUNGLE

path from TEEMO to JINX:
TEEMO to JINX via ADC

path from JINX to SONA:
JINX to SONA via FRIENDS